config/get_mark.sh

```diff
@@ -0,0 +1,12 @@
+set -ex
+
+if [[ $1 == "crawler" ]]; then
+  export TARGET_SCORE=$(head -5 config/target_score.txt | tail -1)
```

Comment:
this is wrong! here you get the mark for pipeline(!!)

Reply:
fixed